css/_nav.scss

```
@@ -0,0 +1,41 @@
+.section-nav {
+  background: #5373fd;
+  padding: 32px;
+}
+
+.section-nav__wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```

використовуй вкладеність селекторів в SCSS
```suggestion
.section-nav {
  background: #5373fd;
  padding: 32px;
  
  &__wrapper {
    display: flex;
    justify-content: space-between;
    align-items: center;
  }
}
```